### PR TITLE
"Print docx1.0 flow"

### PR DESCRIPTION
**Pull request by snehar-nd**

### CODE REVIEW

**Comment by snehar-nd** [comment]
@pallakartheekreddy Please review the PR.

**Comment by snehar-nd** [comment]
@pallakartheekreddy Please merge my PR.

File: src/envVariables.js [hunk 1]
```diff
@@ -1,45 +1,57 @@
...
-        database: process.env.sunbird_program_db_name || 'sunbird_programs',
-        password: process.env.sunbird_program_db_password || 'password',
-        port: process.env.sunbird_program_db_port || 5432,
-        dialect: process.env.sunbird_program_db_dialect || "postgres",
-        logging: false,
-        pool: {
-            max: process.env.sunbird_program_db_pool ? Number(process.env.sunbird_program_db_pool) : 100
-        }
+  // baseURL: process.env.dock_base_url || 'https://vdn.diksha.gov.in',
+  baseURL: process.env.dock_base_url || "https://dock.sunbirded.org",
```
Comment (pallakartheekreddy) [inline src/envVariables.js:3]: @snehar-nd WHy are committing commented code. Please don't add this file if you don't have any changes in this file.

File: src/envVariables.js [hunk 2]
```diff
@@ -1,45 +1,57 @@
...
+  //https://dockstaging.sunbirded.org/
+  //https://dock.preprod.ntp.net.in/
+  //https://dock.sunbirded.org
+  // baseURL: "http://assessment-service:9000/",
+  TOKEN:
+    "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiIyOGJlNzFmZGNlYTM0ZDhkOWJlYzk4NGIwZjdhMTc4YiJ9.wzl9Jw_y2RI0cv6Ls9zOQ23HpvHiKQ_Y3VByXBsZSbE",
```
Comment (pallakartheekreddy) [inline src/envVariables.js:9]: WE SHOULD NOT COMMIT ANY TOKENs.

Comment (snehar-nd) [reply src/envVariables.js:9]: okay.
I need this token, to read the Question set hierarchy API. 
then I should create .env file and keep that token in that .env?

Comment (snehar-nd) [reply src/envVariables.js:9]: @pallakartheekreddy SUNBIRD_PORTAL_API_AUTH_TOKEN

File: src/routes/print.js [hunk 3]
```diff
@@ -46,50 +92,60 @@ async function printDocx(req,res){
   });
 }
 
-async function printPDF(req, res) {
```
Comment (pallakartheekreddy) [inline src/routes/print.js:49]: Please remove commented code

File: src/routes/print.js [hunk 4]
```diff
@@ -46,50 +92,60 @@ async function printDocx(req,res){
...
-      }
-    } else {
-      res.status(404).send({
-        error: errorMsg,
-      });
-    }
-  });
 }
+
+// async function printxml(req,res){
```
Comment (pallakartheekreddy) [inline src/routes/print.js:97]: Please remove commented code

File: src/service/print/dataImporter.js [hunk 5]
```diff
@@ -1,16 +1,18 @@
...
-    this.name = "PDFDataImportError";
+    this.name = "PrintDocxDataImportError";
   }
 }
 
-function getItemsFromItemset(itemsetID,marks) {
+function getItemsFromItemset(itemsetID, marks) {
   let status;
-  const urlItemset = `${envVariables.baseURL}/action/itemset/v3/read/${itemsetID}`;
+  const urlItemset = `${envVariables.LEARNER_SERVICE_URL}itemset/v3/read/${itemsetID}`;
```
Comment (pallakartheekreddy) [inline src/service/print/dataImporter.js:15]: Why are you changing `baseURL` to `LEARNER_SERVICE_URL`

File: src/service/print/printDocx-1.0/dataImporter.js [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+      errormsg:"wrong ID"
+    }
+  })
+};
+const getQuestionSet = async (id) => {
+  const headers = {
+    Authorization: "Bearer " + envVariables.SUNBIRD_PORTAL_API_AUTH_TOKEN
+  };
+  let url;
+  if(envVariables.ASSESSMENT_SERVICE_URL === "http://assessment-service:9000"){
```
Comment (pallakartheekreddy) [inline src/service/print/printDocx-1.0/dataImporter.js:44]: This is not correct way to write code

File: src/routes/print.js (hunk 4 above)
Comment (snehar-nd) [reply src/routes/print.js:97]: removed.

File: src/service/print/dataImporter.js (hunk 5 above)
Comment (snehar-nd) [reply src/service/print/dataImporter.js:15]: As mentioned in the yml file  https://github.com/project-sunbird/sunbird-devops/blob/release-4.6.0-vdn/ansible/roles/kong-api/defaults/main.yml  
Item set read URL will point to this - `upstream_url: "{{ learning_service_url }}/itemset/v3/read"`  , and even I changed the envVariables.js file also.

File: src/service/print/printDocx-1.0/dataImporter.js (hunk 6 above)
Comment (snehar-nd) [reply src/service/print/printDocx-1.0/dataImporter.js:44]: This I have made changes like removed if the condition,

**Comment by snehar-nd** [comment]
Raising new PR with all corrections

**Pull request closed without merging**